test/scripts/checks/commit/commit_check_runner.sh

```diff
@@ -7,7 +7,7 @@ If you want, you can still manually install the pre-commit hook locally by runni
 !!!!!!!!!!!!!!!!!!!!!!!!!!!
 "
 
-  node scripts/precommit_hook.js --ref HEAD~1..HEAD --max-files 200
+  node scripts/precommit_hook.js --ref HEAD --max-files 200
```

Comment:
Hmm, I'm not sure what this goal with this previous ref was, but mind explaining why it needed to change?

Reply:
The argument of `--ref` was ending as argument to a `git diff`, now it ends as argument to a `git show`. The result is identical, the list of files changed by the commit at `HEAD`, but the change makes `--ref` accepting a real ref as per docs:
```
--ref Run checks against any git ref files (example HEAD or <commit_sha>) instead of running against staged ones
```

Reply:
Sounds good, thanks